tests/phpunit/tests/html-api/wpHtmlProcessorMetaTag.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Unit tests covering WP_HTML_Processor META tag handling.
+ *
+ * @package WordPress
+ * @subpackage HTML-API
+ *
+ * @since 6.9
+ *
+ * @group html-api
+ *
+ * @coversDefaultClass WP_HTML_Processor
+ */
+class Tests_HtmlApi_WpHtmlProcessorMetaTag extends WP_UnitTestCase {
+	/**
+	 * Data provider.
+	 */
+	public static function data_supported_meta_tags(): array {
+		return array(
+			'No attributes'                        => array( '<meta>' ),
+			'Unrelated attributes'                 => array( '<meta not-charset="OK">' ),
+			'Boolean charset'                      => array( '<meta charset>' ),
+			'HTTP Equiv: accept'                   => array( '<meta http-equiv="accept" content="">' ),
+			'HTTP Equiv: content-type, no content' => array( '<meta http-equiv="content-type">' ),
+			'Boolean HTTP Equiv'                   => array( '<meta http-equiv content="">' ),
+		);
+	}
+
+	/**
+	 * Ensures that META tags correctly handle encoding confidence.
+	 *
+	 * @ticket 63738
+	 *
+	 * @dataProvider data_supported_meta_tags
+	 */
+	public function test_supported_meta_tag( string $html ) {
+		$html      = '<!DOCTYPE html>' . $html;
+		$processor = new class($html) extends WP_HTML_Processor {
+			public function __construct( $html ) {
+				parent::__construct( $html, parent::CONSTRUCTOR_UNLOCK_CODE );
+			}
+		};
+
+		$this->assertTrue( $processor->next_tag( 'META' ) );
+	}
+
+	/**
+	 * Data provider.
+	 */
+	public function data_unsupported_meta_tags(): array {
```

Comment:
```suggestion
	public static function data_unsupported_meta_tags(): array {
```

Make a `static` method

Reply:
@sirreal ☝️

Reply:
Good spot, thanks: https://github.com/WordPress/wordpress-develop/pull/9318